mythril/analysis/modules/external_calls.py

```diff
@@ -31,8 +34,24 @@ def _analyze_state(state):
     node = state.node
     gas = state.mstate.stack[-1]
     to = state.mstate.stack[-2]
-
+    issues = []
     address = state.get_current_instruction()["address"]
+    annotations = cast(
```

Comment:
Correct me if I'm wrong but it seems to me that this module does not really use/need any annotations.
If so we can remove this code.

Moreover, I don't think different modules should share annotations, they each have different behaviours modifying the annotation. Which can make for some nasty bugs.
If both modules need to use an annotation I think they need separate annotations

Reply:
The data from the computations done in this module is used in state change module(the `user_defined_address`), so i think it might make sense combine these two modules. WDYT?